drivers/at86rf2xx/at86rf2xx_internal.c

```diff
@@ -211,6 +211,25 @@ void at86rf2xx_configure_phy(at86rf2xx_t *dev)
     at86rf2xx_set_state(dev, state);
 }
 
+#ifdef MODULE_AT86RF233
+void at86rf2xx_get_random(at86rf2xx_t *dev, uint8_t *data, const size_t len)
+{
+    for (size_t byteCount = 0; byteCount < len; ++byteCount) {
+        uint8_t rnd = 0;
+        for (uint8_t i = 0; i < 4; ++i) {
+            /* bit 5 and 6 of the AT86RF2XX_REG__PHY_RSSI register contain the RND_VALUE */
+            uint8_t regVal = at86rf2xx_reg_read(dev, AT86RF2XX_REG__PHY_RSSI)
+                             & AT86RF2XX_PHY_RSSI_MASK__RND_VALUE;
```

Comment:
Don't know if this is convention, but I prefer the operators to be trailing the leading line.